pom.xml

```diff
@@ -154,6 +154,7 @@
     <module>java-resourcemanager</module>
     <module>java-retail</module>
     <module>java-run</module>
+    <module>java-samples</module>
```

Comment:
The intent was to use the [same solution used in spring-cloud-gcp](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/main/spring-cloud-gcp-samples/pom.xml#L265-L270), and avoid the need for special CI logic. 

This is what prevents the sample from being deployed:
https://github.com/googleapis/google-cloud-java/pull/10805/files#diff-27d2eee66ae8d7e40703e2670c7107abc6667008967085105fab6f87624aeff9R41-R47

Reply:
> The intent was to use the [same solution used in spring-cloud-gcp](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/main/spring-cloud-gcp-samples/pom.xml#L265-L270), and avoid the need for special CI logic.

+1 to add a little bit more context to this - Without this change or special customizations to the script, we run into the following error:
```
18:33:33:151 [WARNING] 
18:33:33:151 [WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
18:33:33:152 [WARNING] 
18:33:33:152 [WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
18:33:33:152 [WARNING] 
18:33:33:221 [ERROR] [ERROR] Could not find the selected project in the reactor: java-samples/native-image-sample @ 
18:33:33:222 [ERROR] Could not find the selected project in the reactor: java-samples/native-image-sample -> [Help 1]
18:33:33:222 [ERROR] 
18:33:33:223 [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
18:33:33:223 [ERROR] Re-run Maven using the -X switch to enable full debug logging.
```

Reply:
Yeah - removing the module from the parent means needing to `cd` into the sample directory and run `mvn` commands from there. So it'd be a dedicated set of scripts to perform that testing.

Reply:
I see. To add precaution, would you set the versions of the sample modules as "1.0-SNAPSHOT" without the Release Please annotation? That ensures the modules won't be published to Maven Central accidentally.

Reply:
Done, thank you.